src/lib/minions/data/killableMonsters/bosses/misc.ts

```diff
@@ -161,10 +162,8 @@ const killableBosses: KillableMonster[] = [
 		qpRequired: 0,
 		itemInBankBoosts: [
 			{
-				[itemID('Dragon warhammer')]: 10
-			},
-			{
-				[itemID('Elder maul')]: 5
+				[itemID('Dragon warhammer')]: 10,
+				[itemID('Elder maul')]: 12
```

Comment:
fairly certain these should be flipped around to check for the highest boost item first